### ISSUE TITLE
the links of pre-trained checkpoint is unavailable

### ISSUE THREAD

**Issue by Four-Fire** [cpeng93/DiffuseRecon#4]
Hi, it seems like that the links of pre-trained checkpoint is unavailable. 
The first link shows that:
![image](https://user-images.githubusercontent.com/72636550/221563834-01190d51-d3de-4a48-ac1a-41bec634649a.png)
And the second link shows that:
![image](https://user-images.githubusercontent.com/72636550/221563951-247d99f0-aaa7-4d36-8ccf-a689d7d09ab1.png)
Can you update them? Thanks.

**Comment by cpeng93** [comment]
See if this works: https://www.dropbox.com/s/9fk6hdepv2xi4m3/ema_0.9999_150000.pt?dl=0

**Comment by Four-Fire** [comment]
> See if this works: https://www.dropbox.com/s/9fk6hdepv2xi4m3/ema_0.9999_150000.pt?dl=0

This link works! But when I try to run 'image_sample_complex_duo.py', get some errors. It seems like that The model parameter information has changed.
![image](https://user-images.githubusercontent.com/72636550/222313069-8e7f2d57-fb7b-44c1-8645-e8a78a9f5408.png)

**Comment by cpeng93** [comment]
I am not sure how to debug this as the screenshot is only partial, I am guessing that some parameters of the model is not set correctly e.g. --num_channels 128

**Comment by Four-Fire** [comment]
> I am not sure how to debug this as the screenshot is only partial, I am guessing that some parameters of the model is not set correctly e.g. --num_channels 128

OK, I removed my changes and brought the code back to the default state, which means I used the data_process.py and used the image_sample_complex_duo.py to sample. The test data is file1000031.h5. However, the sampled result is not good. 
The result png:
![file1000031_16](https://user-images.githubusercontent.com/72636550/223029344-79f8ca2f-608f-46ba-932d-ec8f6ac43e0b.png)

The reference png:
![16recon](https://user-images.githubusercontent.com/72636550/223009099-496ceba7-d7ee-48b3-8a47-bb6094624547.png)

The sampling code is that you suggested:
`python scripts/image_sample_complex_duo.py --model_path img_space_dual/ema_0.9999_150000.pt --data_path EVAL_PATH \
--image_size 320 --num_channels 128 --num_res_blocks 3 --learn_sigma False --dropout 0.3 --diffusion_steps 4000 \
--noise_schedule cosine --timestep_respacing 100 --save_path test/ --num_samples 1 --batch_size 5`
@cpeng93

**Comment by JustlfC03** [comment]
I have the same question，the  sampled result 31 is not good.

**Comment by JustlfC03** [comment]
> I have the same question，the sampled result 31 is not good.

@cpeng93
